webrtc.html

```diff
@@ -11573,9 +11573,89 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
+              "idp-tls-failure",
+              "idp-token-expired",
+              "idp-token-invalid",
+              "parameters-error",
+              "sctp-failure",
+              "sdp-syntax-error"
+          };</pre> 
+          <table data-link-for="RTCErrorDetailType" data-dfn-for=
+          "RTCErrorDetailType" class="simple">
+            <tbody>
+              <tr>
+                <th colspan="2">Enumeration description</th>
+              </tr>
+              <tr>
+                <td><dfn><code>data-channel-failure</code></dfn></td>
+                <td>The data channel has failed.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>dtls-failure</code></dfn></td>
+                <td>The DTLS negotiation has failed or the connection
+                  has been terminated with a fatal error.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-bad-script-failure</code></dfn></td>
+                <td>The script loaded from the identity provider is not valid
+                  JavaScript or did not implement the correct interfaces.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-execution-failure</code></dfn></td>
+                <td>The identity provider has thrown an exception or
+                  returned a rejected promise.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-load-failure</code></dfn></td>
+                <td>Loading of the IdP URI has failed.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-need-login</code></dfn></td>
+                <td>The identity provider requires the user to login.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-timeout</code></dfn></td>
+                <td>The IdP timer has expired.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-tls-failure</code></dfn></td>
+                <td>The TLS certificate used for the IdP HTTPS connection
+                  is not trusted.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-token-expired</code></dfn></td>
+                <td>The IdP token has expired.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>idp-token-invalid</code></dfn></td>
+                <td>The IdP token is invalid.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>parameters-error</code></dfn></td>
+                <td>The <code>RTCRtpParameters</code> is not valid.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>sctp-failure</code></dfn></td>
+                <td>The SCTP negotiation has failed or the connection
+                  has been terminated with a fatal error.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>sdp-syntax-error</code></dfn></td>
+                <td>The SDP syntax is not valid.</td>
```

Comment:
Added material about sdpLineNumber in "sdp-syntax-error", sctpCauseCode in "sctp-failure", httpRequestStatusCode in "idp-load-failure" and idpLoginUrl in "idp-need-login".